doc/tutorials/navigation2_on_real_turtlebot3.md

```diff
@@ -79,11 +79,11 @@ Launch RVIZ with a pre-defined configuration file.
 
 Now, you should see a shadow of Turtlebot 3 robot model in the center of the plot in Rviz. Click on the Start button (Bottom Left) if you set the auto_start parameter to false.
 
-![enter image description here](https://github.com/mlherd/ros2/blob/master/img/Rviz_start.PNG?raw=true)
+![enter image description here](https://github.com/ros-planning/navigation2/blob/master/doc/tutorials/images/Navigation2_on_real_Turtlebot3/rviz_after_launch_view.png?raw=true)
```

Comment:
this is overkill, you're causing an actual https call for a local asset. In github you can specify a local path `./images/test_image.png`